.gitignore

```diff
@@ -44,3 +44,6 @@ build/
 # Web UI tools
 node_modules
 package-lock.json
+
+# Other
+.vscode/
```

Comment:
There are other IDE related artifacts listed above:
>CMakeLists.txt.user*
qbittorrent.pro.user*

So why not keep them together?
Anyway `Other` isn't good section name for it since since it is clear that it belongs to a more specific type.